libs/core/src/lib/facets/facet/facet.component.html

```diff
@@ -0,0 +1,31 @@
+<ng-container *ngIf="type === 'image'">
+    <ng-container *ngTemplateOutlet="image"></ng-container>
```

Comment:
why didn't you just put `<ng-container *ngTemplateOutlet="customContentTemplate"></ng-container>` here?

Reply:
removed and simplified the template a bit